amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4292,11 +4302,15 @@ definitions:
         type: string
       combineDispositionStepConditions:
         type: boolean
+        default: false
       eligibleOnFirstCompleteEvent:
         type: boolean
+        default: true
       description:
         type: string
-      destroyWithGhosting:
+      destroyNode:
+        type: boolean
+      destroyContent:
```

Comment:
Ah - there's been some misunderstanding here. I suggest deleting these two new properties and updating the possible types below (I'll add a separate comment for that).

Reply:
We have kept only one property named as **retainRecordMetadataAfterDestruction** rather than destroyNode and destroyContent as the end user will find this field more in sync with frontend and it will remove the confusion.